test/project_types/script/commands/enable_test.rb

```diff
@@ -112,34 +92,45 @@ def test_calls_application_enable_with_configuration_properties
             },
           ],
         }
-        Script::Layers::Application::EnableScript.expects(:call).with(
-          ctx: @context,
-          api_key: @api_key,
-          shop_domain: @shop_domain,
-          configuration: expected_configuration,
-          extension_point_type: @ep_type,
-          title: @script_name
-        )
-
-        @context
-          .expects(:puts)
-          .with(@context.message(
-            'script.enable.script_enabled',
-            api_key: @api_key,
-            shop_domain: @shop_domain,
-            type: @ep_type.capitalize,
-            title: @script_name
-          ))
 
-        @context
-          .expects(:puts)
-          .with(@context.message('script.enable.info'))
+        expect_successful_enable(expected_configuration)
 
         capture_io do
           perform_command(config_props: "key1:value1,key2:value2")
         end
       end
 
+      def test_with_config_props_with_spaces
+        expected_configuration = {
+          entries: [
+            {
+              key: "key1",
+              value: "value1",
+            },
+            {
+              key: "key2",
+              value: "with spaces",
+            },
+          ],
+        }
+
+        expect_successful_enable(expected_configuration)
+
+        capture_io do
+          perform_command(config_props: "key1:value1 , key2:with spaces")
+        end
+      end
+
+      def test_should_call_error_handler_when_given_invalid_config_props
+        Script::UI::ErrorHandler
+          .expects(:pretty_print_and_raise)
+          .with { |e| e.is_a?(Errors::InvalidConfigProps) }
```

Comment:
Maybe use the `instance_of` matcher here?

https://mocha.jamesmead.org/Mocha/ParameterMatchers.html#instance_of-instance_method

Reply:
TIL, I like that. Updated it!